code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -342,3 +342,12 @@
 	attribute_requirements = list(
 					PRUDENCE_ATTRIBUTE = 80
 					)
+
+/obj/item/clothing/suit/armor/ego_gear/innocence
+	name = "childhood memories"
+	desc = "In my dreams as child, Peter Pan would reach out a hand for me to hold and take me to Neverland. I had forgotten all of that, until I went into that room."
+	icon_state = "innocence"
+	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 60, BLACK_DAMAGE = 60, PALE_DAMAGE = 10) //140
```

Comment:
This might be too much

Reply:
What do you propose as an alternative here? 10 red and pale really is not actually very good